entry_tasks/46_calculator/main.py

```diff
@@ -0,0 +1,21 @@
+if __name__ == "__main__":
+    a = int(input("Enter first integer: "))
+    b = int(input("Enter second integer: "))
+    op = input('''
+    Select operation:
+    + Add
+    > Greater than
+    or enter another operation
+    ''')
+    if op == '+':
+        if a == 1 and b == 2:
```

Comment:
why do we need to check if those values are equal to 1 or 2? (since it's user's input)